Больше деталей

SystemAgainst opened the pull request:
1. Created lits or array of predefined values
2. Created functions for created author, offer, and locations
3. Created function that generated 10 js-objects

---
:mortar_board: [Больше деталей](https://up.htmlacademy.ru/javascript/27/user/2148431/tasks/9)

:boom: https://htmlacademy-javascript.github.io/2148431-keksobooking-27/5/

SystemAgainst committed:
Getting 10 generated js-objects.

vmolchanov commented:
@SystemAgainst Число комнат, число гостей и цену сгенерируй через функцию `getRandomPositiveInteger `

SystemAgainst committed:
Corrected code

Amount of price, rooms and guests was generated randomly not by certain list of numbers, but by function getRandomPositiveInteger()
